File: Modul 6 Sem 1/Phython/PRAK603-2410817110005-MuhammadRizqanKarim.py
# PRAK603-NIM-Nama.py
n1, n2 = map(int, input("Masukkan n1 dan n2: ").split())
if n1 != n2:
    print("Jumlah tidak sama")
else:
    row1 = list(map(int, input("Masukkan elemen baris pertama: ").split()))
    row2 = list(map(int, input("Masukkan elemen baris kedua: ").split()))
    result = [row1[i] * row2[i] for i in range(n1)]
    print(' '.join(map(str, result)))

File: Modul 6 Sem 1/C/PRAK603-2410817110005-MuhammadRizqanKarim.c
// PRAK603-NIM-Nama.c
#include <stdio.h>

int main() {
    int n1, n2;
    printf("Masukkan n1 dan n2: ");
    scanf("%d %d", &n1, &n2);
    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int row1[n1], row2[n2];
        printf("Masukkan elemen baris pertama: ");
        for (int i = 0; i < n1; i++) {
            scanf("%d", &row1[i]);
        }
        printf("Masukkan elemen baris kedua: ");
        for (int i = 0; i < n2; i++) {
            scanf("%d", &row2[i]);
        }
        for (int i = 0; i < n1; i++) {
            printf("%d ", row1[i] * row2[i]);
        }
        printf("\n");
    }
    return 0;
}
